internal/kuberesource/sets.go

```diff
@@ -442,6 +432,19 @@ func Emojivoto(smMode serviceMeshMode) []any {
 	}
 
 	if smMode == ServiceMeshDisabled {
```

Comment:
We are using @3u13r  images for ServiceMeshIngress, too. If they are modified, they might need the environment variables, but if they are not we should replace them with vanilla ones.

Reply:
I'm actually not sure why we aren't using the upstream image. Maybe because the endpoint is https, but the bot talks http by default?

Reply:
For the vote-bot (part of the "web" image): yes. It's a http client but it must be a https client.

Reply:
So we don't need the ENV configuration there, right?

Reply:
No, we don't need it. For more details see: https://github.com/3u13r/emojivoto/commit/8ba877681c297721cde63eb7be95c98c4c1186ee